### ISSUE TITLE
Support network device

### ISSUE THREAD

**Issue by Jason-Hsu** [cansik/pyrealsense2-macosx#3]
Thanks for your work. Does it support the network RealSense connection? I tried to add `-DBUILD_NETWORK_DEVICE=ON` in [this line](https://github.com/cansik/pyrealsense2-macosx/blob/main/librealsense-python-mac.ps1#L79) of CMake, then tried [net_viewer.py](https://github.com/IntelRealSense/librealsense/blob/master/wrappers/python/examples/net_viewer.py). It will have `ImportError: cannot import name 'pyrealsense2_net' from 'pyrealsense2'` error.

**Comment by cansik** [comment]
Thanks for the issue, I will test it.

**Comment by cansik** [comment]
Yes, there seems to be a problem but there is no error message or something like that. The following thread has the same problems and maybe is connected https://github.com/IntelRealSense/librealsense/issues/9946.
